systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -210,13 +212,13 @@ void testBuildFailsWithWrongChecksumOfArtifact(ExtensionContext extensionContext
 
     @ParallelTest
     void testBuildWithJarTgzAndZip(ExtensionContext extensionContext) {
-        TestStorage testStorage = new TestStorage(extensionContext, clusterOperator.getDeploymentNamespace());
+        TestStorage testStorage = new TestStorage(extensionContext, Constants.TEST_SUITE_NAMESPACE);
```

Comment:
I wonder if the constant shouldn't be part of ResourceManager, it looks strange to me have it in so many places

Reply:
Or maybe directly in Abstract

Reply:
Well, in this case, we do not need to specify such a namespace here. I think this is leftover. By default, we use `Constants.TEST_SUITE_NAMESPACE` in TestStorage if a user does not specify a different namespace.

Reply:
I have removed it from the whole class.